packages/serve/index.ts

```diff
@@ -82,6 +83,9 @@ export default function serve(): Promise<void | Function> {
 			);
 			process.exit(1);
 		}
+		// Switch the dir to project root before executing webpack-dev-server
+		process.chdir(findProjectRoot());
```

Comment:
No, we are not manipulating the cwd through the CLI, this is undefined user behaviour and up to the user themselves. You will need to provide the serve command context property (rootpath), which will be sent to webpack.